FirstProject/assets/new-evaluation.less

```diff
@@ -0,0 +1,161 @@
+//New evaluation less file
+
+.container {
+
+  .candidate_input {
+    margin-left: 95px;
+    border-right: none;
+    border-left: none;
+    border-top: none;
+  }
+
+  .interviewer_input {
+    margin: 0 110px 0 110px;
+    border-right: none;
+    border-left: none;
+    border-top: none;
+
+  }
+
+  .calendar_input {
+    margin: 0 110px 0 110px;
+    border: none;
+  }
+}
+
+.row {
+
+  .logo_section {
+    border-bottom: 2px solid darkgrey;
+    padding-bottom: 15px;
+  }
+}
+
+.text_alignment {
+  text-align: center;
+
+  .logo_image {
+    float:left;
+    width: 12%;
+  }
+}
+
+.level_evaluation {
+  background-color: #03A9F4;
+  margin: auto;
+  width: 62.2%;
+  display: flex;
+
+  .bullets_menu{
+
+    h3{
+      color:white;
+      padding-left: 18px;
+    }
+  }
+}
+
+.container {
+  margin-top:100px;
+
+  .row {
+    width: 89%;
+    margin: 0 auto;
+  }
+
+  .description_section {
+    color: #03A9F4;
+  }
+}
+
+.row_section {
+  border: 2px solid grey;
+  display: flex;
+  margin: auto; width: 62%;
+
+  .bullets_menu {
+    padding: 16px;
+  }
+}
+
+.row_first {
+  margin: 0 auto; width: 89%;
+
+  .description_section {
+    color: #03A9F4;
+  }
+
+  textarea {
+    margin-right: 282px;
+    margin-left: 252px;
+    height:150px;
+    border: 2px solid grey;
+  }
+}
+
+.submit_button_section {
+  width: 73%;
+  margin: 0 auto;
+
+  .submit_button {
+    float: right;
+    margin-top: 30px;
+    padding: 15px 40px;
+    margin-right: 100px;
+  }
+}
+
+.container-fluid {
+  margin-top: 80px;
+}
+
+.language_form {
+  text-align:center;
+
+  .field_section {
+    display:inline;
+    width: 61%;
+  }
+
+  ul {
+    display: inline-flex;
+    text-align: right;
+    width: 97%;
+  }
+}
+
+.container_technical_evaluation {
+  margin-top:100px;
+
+  .row {
+    width: 89%;
+    margin: 0 auto;
+  }
+
+  .description_section {
+    color: #03A9F4;
+  }
+
+  .level_evaluation {
+    background-color: #03A9F4;
+    margin: auto;
+    width: 62.2%;
+    display: flex;
+  }
+}
+
+.evaluation_button {
+  margin-left: 150px;
+}
+
+.top_page_inputs {
+  display: block;
+  width: 1200px;
+  margin: 0 auto;
+  margin-top: 50px;
+
+  .col-md-3 {
```

Comment:
I understand your question.

A more simple way to write it once is margin: 50px auto 0 auto;
It wasn't necessary to put 2 rules.